examples/authorize_and_store.js

```diff
@@ -44,7 +38,7 @@ const ipfs = create({
 async function read_from_ipfs(cid) {
     // Fetch the block (downloads via Bitswap if not local)
     console.log('Trying to get cid: ', cid);
-    const block = await ipfs.block.get(cid);
+    const block = await ipfs.block.get(cid, {timeout: 10000});
```

Comment:
Test should work without this, if not we need to showcase for example where to  `await waitForNewBlock()`

```suggestion
    const block = await ipfs.block.get(cid);
```